### PR TITLE
Issue15 - First User becomes admin

### PR DESCRIPTION
**Pull request by Matt-Lepley**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,11 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  before_save :check_if_first_user
+
+  def check_if_first_user
+    User.any? ? self.admin = false : self.admin = true
```
Comment (kenmazaika) [inline app/models/user.rb]: I'd rework this to be:

```
self.admin = ! User.any?
```

File: test/models/user_test.rb [hunk 2]
```diff
@@ -4,4 +4,15 @@ class UserTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+  test "first user becomes admin" do
+    user = User.create(:email => "test1@example.com", :password => "password", :password_confirmation => "password")
+    assert_equal true, user.admin
```
Comment (kenmazaika) [inline test/models/user_test.rb]: I might change this to be:

```
assert user.admin
```

File: test/models/user_test.rb [hunk 3]
```diff
@@ -4,4 +4,15 @@ class UserTest < ActiveSupport::TestCase
...
+  end
+
+  test "second user does not become admin" do
+    user = User.create(:email => "test1@example.com", :password => "password", :password_confirmation => "password")
+    user2 = User.create(:email => "test2@example.com", :password => "password", :password_confirmation => "password")
+    assert_equal false, user2.admin
```
Comment (kenmazaika) [inline test/models/user_test.rb]: I might change this to be:

```
assert ! user2.admin
```

or

```
refute user2.admin
```

**Comment by Matt-Lepley** [comment]
Good suggestions, refactored!

**Comment by kenmazaika** [comment]
Boom!  Merging!

**Pull request merged**